server/test/views/questiontypes/RadioButtonQuestionRendererTest.java

```diff
@@ -100,4 +103,19 @@ public void render_withAriaLabels() {
     assertThat(result.render().matches(".*fieldset aria-describedby=\"[A-Za-z]{8}-description\".*"))
         .isTrue();
   }
+
+  @Test
+  public void renderWithErrors_andSingleErrorMode_hasAutofocus() {
```

Comment:
question (non-blocking): Why did you leave out a test for the absence of "autofocus" in this test suite? I don't know that you need to include it, just curious about the reasoning.

Reply:
I'm also curious about your overall testing strategy. Why did you add tests for Radio and Checkbox but not others types like Text? (Apologies again if we talked about this and I don't remember 🙈 )